data/container-workload.service

```diff
@@ -3,6 +3,7 @@ Description=Container Workload
 After=opt-rootfs-resolv.service
 After=opt-rootfs-proc.mount
 After=opt-rootfs-sys.mount
+After=hyperstart.service
```

Comment:
This doesn't seem necessary, the workload today doesn't need hyperstart (and in the future this file will probably go away)